### ISSUE TITLE
Remove Rinkeby & Localhost from the network switcher (Metamask & Safari)

### ISSUE THREAD

**Issue by FanFanDAO** [popcorndao/sweet-caramel#1209]
## To-do
Remove Rinkeby & Localhost from the network switcher. This is observed on the Metamask & Safari on mobile only.
[Uploading... RPReplay_Final1662571406.MP4]

**Comment by FanFanDAO** [comment]
@Ajiva-D I still see them under Safari mobile browser.

**Comment by Ajiva-D** [comment]
@FanFanDAO I don't think this can even be tested yet cos the new.popcorndao.finance is down. I think you're testing the live website, and I don't push to the live website. I think this should be in review so that it can be reviewed when the website is up

**Comment by RedVeil** [comment]
Note: has been done but needs to be tagged for prod
